pandas/io/formats/html.py

```diff
@@ -111,20 +123,24 @@ def write_th(self, s, header=False, indent=0, tags=None):
             tags = tags or ""
             tags += 'style="min-width: {colspace};"'.format(colspace=self.fmt.col_space)
 
-        return self._write_cell(s, kind="th", indent=indent, tags=tags)
+        self._write_cell(s, kind="th", indent=indent, tags=tags)
 
-    def write_td(self, s, indent=0, tags=None):
-        return self._write_cell(s, kind="td", indent=indent, tags=tags)
+    def write_td(self, s: str, indent: int = 0, tags: Optional[str] = None) -> None:
+        self._write_cell(s, kind="td", indent=indent, tags=tags)
 
-    def _write_cell(self, s, kind="td", indent=0, tags=None):
+    def _write_cell(
+        self, s: str, kind: str = "td", indent: int = 0, tags: Optional[str] = None
+    ) -> None:
         if tags is not None:
             start_tag = "<{kind} {tags}>".format(kind=kind, tags=tags)
         else:
             start_tag = "<{kind}>".format(kind=kind)
 
         if self.escape:
             # escape & first to prevent double escaping of &
-            esc = OrderedDict([("&", r"&amp;"), ("<", r"&lt;"), (">", r"&gt;")])
+            esc = OrderedDict(
+                [("&", r"&amp;"), ("<", r"&lt;"), (">", r"&gt;")]
+            )  # type: Union[OrderedDict[str, str], Dict]
```

Comment:
I think also OK to just make this an ordereddict all of the time; simplifies annotation and code

Reply:
i was getting `pandas\io\formats\html.py:145: error: Incompatible types in assignment (expression has type "Dict[<nothing>, <nothing>]", variable has type "OrderedDict[str, str]")`

Reply:
Yea I so meant you *could* change the dict on line 145 to be an OrderedDict to match the signature here. I think OK to refactor code if it simplifies annotations and keeps same functionality, as there probably helps readability too

Though this is obviously fine as is too!